cmd/minishift/cmd/hostfolder/add.go

```diff
@@ -59,10 +59,10 @@ var hostfolderAddCmd = &cobra.Command{
 
 func init() {
 	HostfolderCmd.AddCommand(hostfolderAddCmd)
-	hostfolderAddCmd.Flags().BoolVarP(&instanceOnly, "instance-only", "", false, "Define host folder only for the current cluster instance.")
+	hostfolderAddCmd.Flags().BoolVarP(&instanceOnly, "instance-only", "", false, "Defines the host folder only for the current cluster.")
 
 	// Windows-only
 	if runtime.GOOS == "windows" {
-		hostfolderAddCmd.Flags().BoolVarP(&usersShare, "users-share", "", false, "Define host folder for the Users share on a Windows host.")
+		hostfolderAddCmd.Flags().BoolVarP(&usersShare, "users-share", "", false, "Defines the shared `Users` folder as the host folder on a Windows host.")
```

Comment:
> The single quotes are for the parsed docs, not for being copied/pasted along with the folder name.

Not sure what you are saying here, but my guess is that you are adding it, because in the docs the backticks then have a special meaning. If this is the case, a big -1.

This help messages are primarily for the console. It just happens to be that we also generate markdown/asciidoc from it. The primary output we are targeting it the console.

For me as a console user I would wonder what the backticks are about. They are not common practice and they are potentially in the way, for example when I want to cut&paste the message.

So let's please remove them.